Makefile

```diff
@@ -23,6 +23,9 @@ $(module)/talk.html: $(module)/talk.rst
 
 install: $(BIN)
 	install $(BIN) $(PREFIX)/bin
+	@mkdir -p /etc/bash_completion.d
```

Comment:
True. But I basically copied it from https://github.com/tj/git-extras/blob/master/Makefile#L44